gbdk-lib/examples/cross-platform/platformer_template/src/main.c

```diff
@@ -0,0 +1,87 @@
+#include <gbdk/platform.h>
+#include <stdint.h>
+#include "player.h"
+#include "common.h"
+#include "level.h"
+#include "Camera.h"
```

Comment:
Capitalization here causes a case sensitivity error on Linux. Should be "camera.h"
src/main.c:6:10: fatal error: Camera.h: No such file or directory
    6 | #include "Camera.h"